sdks/web-sdk/src/lib/pages/CheckDocument.svelte

```diff
@@ -44,6 +45,14 @@
         on:click={() => goToPrevStep(currentStepId, $configuration, $currentStepId)}
       />
     {/if}
+    {#if element.type === Elements.IconCloseButton}
+      <IconCloseButton
+        configuration={element.props}
+        on:click={() => {
+          sendButtonClickEvent('close', { status: 'document_collection' }, $appState, true);
```

Comment:
@sFiFty Please use enums instead of a string. Also, it's a good opportunity to change "document_collection" name since it basically indicates the end user left in the middle of the flow, and documet_collection sound like a specific phase of the flow - it will make more sense to call it "data_collection"